### ISSUE TITLE
Getting TypeError exception in Python 3.5

### ISSUE THREAD

**Issue by FrankC01** [kigawas/eciespy#11]
Environment:
 (docker) Python 3.5.1 in ubuntu:xenial

packages:
```
asn1crypto (0.24.0)
certifi (2018.8.24)
cffi (1.11.5)
chardet (3.0.4)
coincurve (8.0.2)
colorlog (3.1.4)
cryptography (2.3.1)
cytoolz (0.9.0.1)
eciespy (0.1.2)
eth-hash (0.1.4)
eth-keys (0.2.0b3)
eth-typing (1.1.0)
eth-utils (1.2.0)
idna (2.7)
pip (8.1.1)
protobuf (3.6.1)
pycparser (2.18)
pycryptodomex (3.6.6)
pysha3 (1.0.2)
PyYAML (3.13)
pyzmq (17.1.2)
requests (2.19.1)
sawtooth-sdk (1.0.5)
sawtooth-signing (1.0.5)
secp256k1 (0.13.2)
setuptools (20.7.0)
six (1.11.0)
toml (0.9.4)
toolz (0.9.0)
urllib3 (1.23)
wheel (0.29.0)
```

Trace:
```
File "/project/hashblock-exchange/modules/state.py", line 19, in <module>
hashblock-rest        |     from ecies import aes_encrypt, aes_decrypt
hashblock-rest        |   File "/usr/local/lib/python3.5/dist-packages/ecies/__init__.py", line 2, in <module>
hashblock-rest        |     from ecies.utils import generate_key, hex2prv, hex2pub, derive, aes_encrypt, aes_decrypt
hashblock-rest        |   File "/usr/local/lib/python3.5/dist-packages/ecies/utils.py", line 8, in <module>
hashblock-rest        |     from eth_keys import keys
hashblock-rest        |   File "/usr/local/lib/python3.5/dist-packages/eth_keys/__init__.py", line 15, in <module>
hashblock-rest        |     from .main import (  # noqa: F401
hashblock-rest        |   File "/usr/local/lib/python3.5/dist-packages/eth_keys/main.py", line 3, in <module>
hashblock-rest        |     from eth_keys.datatypes import (
hashblock-rest        |   File "/usr/local/lib/python3.5/dist-packages/eth_keys/datatypes.py", line 8, in <module>
hashblock-rest        |     from eth_utils import (
hashblock-rest        |   File "/usr/local/lib/python3.5/dist-packages/eth_utils/__init__.py", line 24, in <module>
hashblock-rest        |     from .applicators import (  # noqa: F401
hashblock-rest        |   File "/usr/local/lib/python3.5/dist-packages/eth_utils/applicators.py", line 43, in <module>
hashblock-rest        |     def combine_argument_formatters(*formatters: List[Callable[..., Any]]) -> Formatters:
hashblock-rest        |   File "/usr/lib/python3.5/typing.py", line 1025, in __getitem__
hashblock-rest        |     tvars = _type_vars(params)
hashblock-rest        |   File "/usr/lib/python3.5/typing.py", line 284, in _type_vars
hashblock-rest        |     _get_type_vars(types, tvars)
hashblock-rest        |   File "/usr/lib/python3.5/typing.py", line 279, in _get_type_vars
hashblock-rest        |     t._get_type_vars(tvars)
hashblock-rest        |   File "/usr/lib/python3.5/typing.py", line 786, in _get_type_vars
hashblock-rest        |     _get_type_vars(self.__args__, tvars)
hashblock-rest        |   File "/usr/lib/python3.5/typing.py", line 277, in _get_type_vars
hashblock-rest        |     for t in types:
hashblock-rest        | TypeError: 'ellipsis' object is not iterable
```

**Comment by kigawas** [comment]
Thanks for reporting.

This is a bug which was fixed on Python 3.5.3 and above. 

Ref:
https://github.com/altair-viz/altair/issues/972#issuecomment-400311001

**Comment by FrankC01** [comment]
Thanks for the information, I'll see about moving up to 3.5.3 (or better I suppose).
